code/modules/crafting/recipes.dm

```diff
@@ -218,6 +218,21 @@
 	..()
 	blacklist += subtypesof(/obj/item/gun/energy/laser)
 
+/datum/crafting_recipe/silencer
+	name = "u-ION Silencer"
+	tools = list(TOOL_SCREWDRIVER, TOOL_WIRECUTTER)
+	result = list(/obj/item/gun/energy/disabler/silencer)
+	reqs = list(/obj/item/gun/energy/gun = 1,
+				/obj/item/stack/cable_coil = 5,
+				/obj/item/weaponcrafting/gunkit/silencer = 1)
```

Comment:
(design) I feel like making it out of a disabler would make more sense.

Reply:
Up for trying that

Reply:
Done!